.ci/gitlab/common.yml

```diff
@@ -1,5 +1,5 @@
 .common:
-  image: ghcr.io/clash-lang/clash-ci-$GHC_VERSION:2022-02-02
+  image: ghcr.io/clash-lang/clash-ci-$GHC_VERSION:2022-05-10
```

Comment:
There are other dates to change, grep for them in `.ci/` and `.github/`

Reply:
Ah okay, thanks!